src/Symfony/Component/VarDumper/Tests/Fixtures/Twig.php

```diff
@@ -26,14 +26,11 @@ public function getTemplateName()
 
     public function getDebugInfo()
     {
-        return array (19 => 2);
+        return array(19 => 2);
     }
 
-    public function getSource()
+    public function getSourceContext()
     {
-        return "   foo bar
-     twig source
-
-";
+        return new Twig_Source("   foo bar\n     twig source\n\n", 'foo.twig', 'foo.twig');
```

Comment:
well, it would avoid issues when refactoring it, reintroducing the bugs I spotted during the review